### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by louiskram**
Hey, this is the german translation. Note that I used the formal adressation of the reader ("Sie"), instead of the informal one ("Du". 
Please also note the comment in line 83

### CODE REVIEW

**Comment by Frooodle** [comment]
Awesome thanks for this!
As for comment L83 I chose to use
pageOrderPrompt in its place
so its using the

pageOrderPrompt=Seitenreihenfolge (Geben Sie eine durch Komma getrennte Liste von Seitenzahlen ein):

File: src/main/resources/messages_de_DE.properties [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+
+#Change permissions
+permissions.title=Berechtigungen ändern
+permissions.header=Berechtigungen ändern
+permissions.warning=Achtung: Damit diese Berechtigungen nicht geändert werden können, wird empfohlen, sie über die "Passwort hinzufügen"-Seite mit einem Passwort zu versehen
+permissions.selectText.1=Das zu ändernde PDF auswählen
+permissions.selectText.2=Zu setzende Berechtigungen
+permissions.selectText.3=Das zusammensetzen des PDFs verhindern
+permissions.selectText.4=Prevent content extraction
```
Comment (Frooodle) [inline src/main/resources/messages_de_DE.properties]: permissions.selectText.4=Prevent content extraction
was this missed?

File: src/main/resources/messages_de_DE.properties [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+#merge
+imageToPDF.title=Image to PDF
+imageToPDF.header=Image to PDF
+imageToPDF.submit=Convert
+
+#pdfToImage
+pdfToImage.title=PDF to Image
+pdfToImage.header=PDF to Image
+pdfToImage.selectText=Image Format
+pdfToImage.submit=Convert
```
Comment (Frooodle) [inline src/main/resources/messages_de_DE.properties]: Do these need anything?

**Comment by louiskram** [comment]
> Awesome thanks for this! As for comment L83 I chose to use pageOrderPrompt in its place so its using the pageOrderPrompt=Seitenreihenfolge (Geben Sie eine durch Komma getrennte Liste von Seitenzahlen ein):
> 
> This is because it seemed like a prompt that would appear elsewhere in future

I see, that makes sense! Then you can omit line 83

**Comment by Frooodle** [comment]
Hi louiskram,

We are updating Stirling-PDF's license to MPL 2.0 (Mozilla Public License Version 2.0) to continue fostering our open-source commitment while introducing more flexible usage of the project.

### What We Need From You:
In order to transition to the MPL 2.0 license, we need the explicit consent of all our contributors. 
1. If you're comfortable relicensing your contributions for this specific PR under MPL 2.0, please respond with:
`
I, louiskram, grant permission to relicense my contributions from this PR to Stirling-PDF under the Mozilla Public License 2.0. 
`
2. If you'd prefer to give blanket permission for all of your contributions to Stirling-PDF (be it through pull requests, commits, or any other form of contribution) rather than on a PR-by-PR basis, please respond with:
`
I, louiskram, grant permission to relicense all my contributions to Stirling-PDF under the Mozilla Public License 2.0.
`

### What does this do and why?
As your old code was contributed under a different license it legally cannot move to a new license without your permission, and since we dont want to lose your code we wanted to reach out. Remember this project is remaining open-source!.

### Deadline:
If we dont receive a response, or if you decline, unfortunately, we will have to remove your contributions from Stirling-PDF before the license change, which is scheduled  in coming months.

### Thank You! 
Your contributions are invaluable, and we hope to continue having them in Stirling-PDF under the new license. Feel free to ask if you have any questions or concerns about this change.

Best,
Frooodle
Stirling-PDF Maintainer

**Comment by Frooodle** [comment]
@louiskram  Any reply otherwise your commit will be removed!

**Comment by louiskram** [comment]
> @louiskram Any reply otherwise your commit will be removed!

I, louiskram, grant permission to relicense my contributions from this PR to Stirling-PDF under the Mozilla Public License 2.0.

**Pull request merged**